active/AnimatedIcon/AnimatedIconSpec.md

```diff
@@ -0,0 +1,894 @@
+tutorial for more instructions. 
+
+## IAnimatedVisualSource2 interface
+
+Provides a Composition `Visual` that can be animated.
+An object that implements this interface can be used as the Source property of an AnimatedIcon.
+
+Call the TryCreateAnimatedVisual method to retrieve an
+[IAnimatedVisual](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.IAnimatedVisual).
+`IAnimatedVisual` can then be used to get a
+Composition [Visual](https://docs.microsoft.com/uwp/api/Windows.UI.Composition.Visual),
+which can be added to a XAML element tree using the methods of
+[ElementCompositionPreview](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Hosting.ElementCompositionPreview).
+
+Use the `SetColorProperty` method to set a color of the animated visual,
+and the `Markers` property to find named positions in the animation timeline.
+
+_Spec note: `IAnimatedVisualSource2` is like `IAnimatedVisualSource` in purpose and use,
+but there is no derivation because the '2' interface does not derive from
+[IDynamicAnimatedVisualSource](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.IDynamicAnimatedVisualSource)._
+
+This example displays and animates an object implementing IAnimatedVisualSource2:
+
+```cpp
+void AddVisualAndShowStartAnimation(
+    const winrt::Border& element,
+    const winrt::IAnimatedVisualSource2 source,
+    const winrt::hstring& initialState,
+    const winrt::hstring& steadyState,
+    const winrt::Color& themeColor )
+{
+    winrt::IAnimatedVisual animatedVisual = source.TryCreateAnimatedIconVisual();
+
+    // Find the positions in the animation of the two states
+    auto const markers = source.Markers())
+    auto const fromProgress = static_cast<float>(markers.Lookup(initialState));
+    auto const toProgress = static_cast<float>(markers.Lookup(steadyState));
+
+    // Set the theme color as the animated visual's foreground.
+    source.SetColorProperty("Foreground", themeColor);
+
+    // Helper that uses TryCreateAnimatedVisual to add the source to the XAML element tree.
+    // See IAnimatedVisualSource.TryCreateAnimatedVisual for more information.
+    AddVisualToElement(element, animatedVisual);
+
+    // Helper to play an animation.
+    // See IAnimatedVisual for more information.
+    PlaySegment(animatedVisual.RootVisual(), fromProgress, toProgress);
+}
+```
+
+## Other IAnimatedVisualSource2 interface members
+
+| Name | Description |
+| :-| :-|
+| Markers | Provides a mapping from marker names to positions in the animation. The names are defined by the animation. |
+| SetColorProperty | Sets a color for the animated visual. The property names are defined by the animation. |
+
+## AnimatedAcceptVisualSource class
+
+This class can be used to show an animation for an Accept operation.
+This can be used as the Source of an AnimatedIcon and is used by the XAML CheckBox control.
+
+This class supports markers that can be driven by an AnimatedIcon.State:
```

Comment:
Shouldn't we be documentation the *states*, not the *transitions*? The transitions are autogenerated from the states, and the system looks for a marker according to the usual algorithm. As documented here, it means that if you create a marker called "NormalOn", it will never be used. But really, it will, to establish the initial state or if there is no applicable transition from the previous state.

States:
* NormalOff
* PointerOverOff
* PressedOff
* NormalOn
* PointerOverOn
* PressedOn